ViewLayer/ViewContainer.h

```diff
@@ -2,17 +2,21 @@
 
 #include <QScopedPointer>
 
-class I_DisplayDashboardUI;
+enum Mode { RACE, DISPLAY };
```

Comment:
Actually, now that I look at my code, isn't the namespace better? Since the enum is used in the EpsilonDashboard class without having a ViewContainer object instantiated.